static/app/components/searchQueryBuilder/tokens/filter/valueListBox.tsx

```diff
@@ -99,7 +99,7 @@ export function ValueListBox<T extends SelectOptionOrSectionWithKey<string>>({
     (acc, item) => acc + (itemIsSection(item) ? item.options.length : 1),
     0
   );
-  const anyItemsShowing = totalOptions > hiddenOptions.size;
+  const anyItemsShowing = totalOptions >= hiddenOptions.size;
```

Comment:
Why does this need to change to `>=`?

Reply:
Okay, I've gone through and fixed this logic, as well as some other bugs that I found with ask seer, and was able to remove the `>=` logic that I had added in.